mcstatus/server.py

```diff
@@ -157,7 +169,7 @@ def query(self, *, tries: int = 3) -> QueryResponse:
         :return: Query information in a :class:`~mcstatus.querier.QueryResponse` instance.
```

Comment:
Why not just add a `port` argument to the query function?

Reply:
There was some discussion about this in #996. Personally, I am not opposed to that, but it does feel odd to not store the query port as initial information for the server. Query port feels like information that should only need to be provided once.